status-im/status-react
# Introduce OFFLINE_INBOX_MANY_ENABLED flag for perf quickfix

Workaround for https://github.com/status-im/status-react/issues/2852 to unblock people from running app (it prevented @yenda from attending test session e.g. IIRC).

Since offline inboxing and desktop were introduced, we went from testing with ~10 messages to ~1000 messages. This surfaced inefficiencies such as this issue. It also means someone who joins a public channel such as #status will experience significant lag when signing in. To remedy this, we put offline inboxing for public/group chats under a flag until the underlying perf issue has been resolved.

### Testing notes (optional):
- Make sure offline inboxing doesn't work for public / group chat
- Check if perf issues are still as bad as #2852 when lots of messages to #status in Desktop e.g. 

This might require setting up a new account to get a fresh state of messages. Otherwise you are likely to hit perf issue due to old messages.

status: ready
